src/Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -213,6 +213,18 @@
                         <FontIcon FontFamily="{StaticResource RecycleBinIcons}" Glyph="&#xEF88;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="RunWithPowerShellButton"
+                    Command="{x:Bind ViewModel.RunWithPowerShellCommand, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.IsPowerShellScript, Mode=OneWay}"
+                    Label="{helpers:ResourceString Name=RunWithPowerShell}"
```

Comment:
Can you add this string to the en-us resource file (we'll generate the the strings in the other files later one)?